beekeeper-scheduler-apiary/src/main/java/com/expediagroup/beekeeper/scheduler/apiary/generator/ExpiredHousekeepingMetadataGenerator.java

```diff
@@ -45,8 +48,10 @@
 
 public class ExpiredHousekeepingMetadataGenerator implements HousekeepingEntityGenerator {
 
+  private static final Logger log = LoggerFactory.getLogger(UnreferencedHousekeepingPathGenerator.class);
```

Comment:
This should be `ExpiredHousekeepingMetadataGenerator.class`

Reply:
Niiiice! You've just earned your "hawk eyes" badge @JayGreeeen!